Awesome books: plain JavaScript with objects

fbclh opened the pull request:
**Project requirements:**
- Create a collection that keeps a list of books 
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection 
- Display all books saved in the collection in the top part of the page.
- When a user clicks the "Add" button, a ​new book is added to the collection.
- When a user clicks the "Add" button, the new book is displayed in the page. 
- When a user clicks the "Remove" button, the correct book is removed from the collection.
- When a user clicks the "Add" button, the correct book disappears from the page..
- Make sure that data is preserved in the browser's memory by using [localStorage].

medaminedev66 commented on README.md:
```diff
@@ -1 +1,57 @@
+
+<!-- ### Deployment -->
+
+## Authors
+
+- GitHub: [github.com/fbclh](https://github.com/fbclh)
+- LinkedIn: [linkedin.com/in/fbclh](https://www.linkedin.com/in/fbclh)
```
- [ ] I see that two developers are working on this project. Kindly add the second developer info to the readme file.

medaminedev66 reviewed (changes requested):
Hi @fbclh and @franklinben23 😃 , :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️ The data is preserved in the localStorage after adding a book and removing a book.

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

fbclh commented:
Hello there thanks for your time in reviewing our project, based on your review I have made the necessary changes:

- Fixed broken demo link.
- Added the link to the other dev working on the project.

medaminedev66 reviewed (approved):
Hi @fbclh and @franklinben23 😃 , 

## Status: Approved
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
